streams/src/main/java/org/apache/kafka/streams/processor/internals/StreamThread.java

```diff
@@ -776,9 +776,10 @@ void runOnce() {
 
                 log.debug("{} punctuators ran.", punctuated);
 
+                final long beforeCommitTs = now;
```

Comment:
such a small nit that I don't think it's worth pushing a fix and restarting the build, but should this be `beforeCommitMs` rather than `Ts`?

Reply:
Happy to address.